src/content/tutorial/wrapping-up/wrapping-up.mdx

```diff
@@ -57,6 +57,6 @@ Submit your completed IBM Digital Badge application by July 30, 2019, and receiv
 
 ## Application
 
-Complete the [application form]() to earn the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+Complete the [application form](https://www.surveygizmo.com/s3/5075530/Dev-Essentials-Badge-Application) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

Comment:
@kellychurchill the link is here.

Reply:
👍 I may have looked at the preview before it rebuilt.